Source/SampleGame426/Private/UI/Base/ActivatableWidget.cpp

```diff
@@ -61,6 +61,17 @@ void UActivatableWidget::DeactivateWidget()
 	}
 }
 
+bool UActivatableWidget::Initialize()
+{
+	const bool bResult = Super::Initialize();
+
+#if UE_5_1_OR_LATER
+	bWidgetInitialized = true;
```

Comment:
Why don't you use the same name `bInitialized`?

Reply:
5.1 or later, it becomes private